package/crds/network.equinix.jet.crossplane.io_sshusers.yaml

```diff
@@ -273,7 +273,7 @@ spec:
                   id:
                     type: string
                   uuid:
-                    description: SSH user unique identifier
+                    description: SSH user unique identifier. SSH user unique identifier
```

Comment:
Many of the descriptions are now duplicated. Reporting this upstream to upjet.

https://github.com/upbound/upjet/issues/155#issuecomment-1455132651